utils/network.go

```diff
@@ -340,3 +345,41 @@ func writeProcSys(path, value string) error {
 	}
 	return err
 }
+
+// Generate a mac address from the passed in string
+func genLocalMacAddr(src string) net.HardwareAddr {
+	// The 2nd bit set of the first octet makes this MAC address a "locally administered address"
+	// https://en.wikipedia.org/wiki/MAC_address
+	macString := "02"
+
+	srcBytes := strings.Split(src, "")
+	srcLen := len(srcBytes)
+	for i, j := 0, 0; i < 10; i++ {
+		if i%2 == 0 {
+			macString += ":"
+		}
+
+		var m string
+		m = ""
+		// Go through the next bytes until a hex one is found
+		for ; j < srcLen && m == ""; j++ {
+			// Only use the byte if the decode succeeds
+			if _, err := hex.DecodeString("0" + srcBytes[j]); err == nil {
+				m = srcBytes[j]
+			} else {
+				logrus.Infof("Convert %s failed with %v", srcBytes[j], err)
```

Comment:
looks like the import is `log`, not `logrus`.

Reply:
I don't think it is worth while to log that out, that was a debug thing that I forgot to remove.  Any problem with just dropping that?

Reply:
nope, that's fine.